Doc/Tutorial/chapter_graphics.tex

```diff
@@ -593,15 +627,59 @@ \subsection{Multiple tracks}
 in Proux \textit{et al.}, they have drawn some unannotated genes.
 
 \begin{minted}{python}
-from reportlab.lib.colors import red, grey, orange, green, brown, blue, lightblue, purple
-
-A_colors = [red]*5 + [grey]*7 + [orange]*2 + [grey]*2 + [orange] + [grey]*11 + [green]*4 \
-         + [grey] + [green]*2 + [grey, green] + [brown]*5 + [blue]*4 + [lightblue]*5 \
-         + [grey, lightblue] + [purple]*2 + [grey]
-B_colors = [red]*6 + [grey]*8 + [orange]*2 + [grey] + [orange] + [grey]*21 + [green]*5 \
-         + [grey] + [brown]*4 + [blue]*3 + [lightblue]*3 + [grey]*5 + [purple]*2
-C_colors = [grey]*30 + [green]*5 + [brown]*4 + [blue]*2 + [grey, blue] + [lightblue]*2 \
-         + [grey]*5
+from reportlab.lib.colors import (
+    red,
+    grey,
+    orange,
+    green,
+    brown,
+    blue,
+    lightblue,
+    purple,
+)
+
+A_colors = (
+    [red] * 5
+    + [grey] * 7
+    + [orange] * 2
+    + [grey] * 2
+    + [orange]
+    + [grey] * 11
+    + [green] * 4
+    + [grey]
+    + [green] * 2
+    + [grey, green]
+    + [brown] * 5
+    + [blue] * 4
+    + [lightblue] * 5
+    + [grey, lightblue]
+    + [purple] * 2
+    + [grey]
+)
+B_colors = (
+    [red] * 6
+    + [grey] * 8
+    + [orange] * 2
+    + [grey]
+    + [orange]
+    + [grey] * 21
+    + [green] * 5
+    + [grey]
+    + [brown] * 4
+    + [blue] * 3
+    + [lightblue] * 3
+    + [grey] * 5
+    + [purple] * 2
+)
+C_colors = (
+    [grey] * 30
+    + [green] * 5
+    + [brown] * 4
+    + [blue] * 2
+    + [grey, blue]
+    + [lightblue] * 2
+    + [grey] * 5
+)
```

Comment:
Cross reference discussion on #2373 which affect the complete copy of this script under ``Doc/examples/``